src/App.tsx

```diff
@@ -17,9 +17,6 @@ import { HomepageMetaTags } from "./components/HomepageMetaTags/HomepageMetaTags
 function App() {
   return (
     <>
-      Kansas City Homeless Union (KCHU) is a union of unhoused Kansas Citians,
-      actively seeking equality and equity, and fighting against the system that
-      perpetuates and sustains homelessness.
```

Comment:
This text seemed to not be visible from the UI, and was being rendered under the nav bar.